auth-server/src/services/auth.ts

```diff
@@ -338,9 +374,14 @@ export default class AuthService {
 			throw new ForbiddenError(messages.ADDRESS_LINKING_FAILED);
 		}
 
+		const otherAddresses = await Address
```

Comment:
We should add comments along the way, this would ease the review and future us to understand what this is doing, it's not entirely clear unless you look at it carefully.

Reply:
done